moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -72,9 +72,9 @@ const ComponentForm = ({
     control,
     watch,
     setValue,
-    formState: { isValid },
+    formState: { isDirty, isValid },
```

Comment:
We couldn't use `isDirty` reliably before because the custom hook that set initial values was making the form dirty. Using `isDirty` (has a field changed from its default or initial value) along with `isValid` helps us keep the submit button inactive unless an actual change has been made.